Source/WebKit/WebProcess/WebPage/WebFoundTextRangeController.cpp

```diff
@@ -367,7 +367,7 @@ void WebFoundTextRangeController::drawRect(WebCore::PageOverlay&, WebCore::Graph
     for (auto& path : foundFramePaths)
         graphicsContext.fillPath(path);
 
-    if (m_textIndicator && !m_textIndicator->selectionRectInRootViewCoordinates().isEmpty()) {
+    if (m_textIndicator && (!m_textIndicator->selectionRectInRootViewCoordinates().isEmpty() || !m_textIndicator->textBoundingRectInRootViewCoordinates().isEmpty())) {
```

Comment:
I don't understand the existing check, and/or am not sure why we're not just replacing it? Nothing else in this block consults the selection rect...

Reply:
Check was added in https://commits.webkit.org/259857@main to mitigate this issue in a different way. This added code makes it unnecessary